string-manipulation-1/src/is-vowel.js

```diff
@@ -0,0 +1,19 @@
+/* exported isVowel */
+
+function isVowel(character) {
+  if (character === 'I' || character === 'E') {
+    return true;
+  } else if (character === 'i' || character === 'e') {
+    return true;
+  } else if (character === 'O' || character === 'U') {
+    return true;
+  } else if (character === 'o' || character === 'u') {
+    return true;
+  } else if (character === 'a') {
+    return true;
+  } else if (character === 'A') {
+    return true;
+  } else {
+    return false;
+  }
```

Comment:
We haven't touched on this in class, but you can also use something called a `switch` statement here. Additional reading [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/switch) if you're interested
```
function isVowel(character) {
  switch (character.toLowerCase()) {
    case 'a':
    case 'e':
    case 'i':
    case 'o':
    case 'u':
      return true;
    default:
      return false;
  }
}
```